File: c#/src/Docmost.Sdk/Services/CommentResolutionApi.cs
using Docmost.Sdk.Http;
using Docmost.Sdk.Models;
using System.Net.Http.Headers;
using System.Net.Http.Json;
using System.Text.Json;

namespace Docmost.Sdk.Services;

public sealed class CommentResolutionApi(DocmostHttpClient http)
{
    private readonly DocmostHttpClient _http = http;

    public async Task<ApiResponse<JsonElement?>> ResolveCommentAsync(ResolveCommentDto request, CancellationToken cancellationToken = default)
    {
        return await _http.PostAsync<JsonElement?>("comments/resolve", request, cancellationToken).ConfigureAwait(false);
    }

}

File: typescript/src/services/comment-resolution-api.ts
/** Generated API client. */
import type { ApiResponse } from '../types.js';
import type { DocmostHttpClient } from '../http-client.js';
import type { ResolveCommentDto } from '../models/index.js';


export class CommentResolutionApi {
  constructor(private readonly http: DocmostHttpClient) {}

  async resolveComment(request: ResolveCommentDto): Promise<ApiResponse<unknown>> {
    return this.http.post(`comments/resolve`, request);
  }

}
